websocketaf-lwt.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@reason-native-web/websocketaf-lwt",
+  "version": "1.0.0",
+  "license": "BSD-3-clause",
+  "homepage": "https://github.com/reason-native-web/vendor",
+  "bugs": { "url": "https://github.com/reason-native-web/vendor/issues" },
+  "esy": {
+    "build": "dune build --only-packages=websocketaf-lwt --root=./websocketaf --profile=release -j 4",
+    "install": "esy-installer #{self.target_dir / 'default' / 'websocketaf-lwt.install'}"
+  },
+  "files": ["websocketaf"],
+  "dependencies": {
+    "ocaml": ">=4.3.0",
+    "@opam/dune": ">=1.7.0",
```

Comment:
I'm wondering how we should handle transitive deps inside our repo. If `weboscketaf` already depends on `@opam/dune` or `@opam/base64` should we still list it here? Probably if it's different projects entirely (like websocketaf vs morph) we should to help the resolution algorithm but in the case of the same project maybe it's better to skip them?

Reply:
I just try to copy what's in the opam file, but maybe we could open a issue for discussion here?

Reply:
Yes.